### PR TITLE
Update MVC templates with 2fa and clean up UI

### PR DESCRIPTION
**Pull request by danroth27**
@DamianEdwards @phenning @HaoK @blowdart Please review

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Models/ManageViewModels/IndexViewModel.cs [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Identity;
 
 namespace Company.WebApplication1.Models.ManageViewModels
 {
     public class IndexViewModel
     {
-        public bool HasPassword { get; set; }
+        public string Username { get; set; }
```
Comment (HaoK) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Models/ManageViewModels/IndexViewModel.cs]: Shouldn't need this, should just use User.Identity.Name

Comment (danroth27) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Models/ManageViewModels/IndexViewModel.cs]: I don't think it hurts to be explicit in the view model about what's going to be rendered in the view.

Comment (phenning) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Models/ManageViewModels/IndexViewModel.cs]: @dotnet-bot please test Debian8.2 x64 Debug

**Comment by phenning** [comment]
@dotnet-bot test Debian8.2 x64 Debug Build please

**Comment by phenning** [comment]
@dotnet-bot test Debian8.2 x64 Debug Build please

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Startup.cs [hunk 2]
```diff
@@ -5,7 +5,6 @@
 #if (OrganizationalAuth || IndividualB2CAuth)
```
Comment (phenning) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Startup.cs]: using Microsoft.AspNetCore.Http is not referenced in Startup.cs. Although this was the case before you change

**Comment by phenning** [comment]
@dotnet-bot test Ubuntu16.10 x64 Debug Build please

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Models/ManageViewModels/IndexViewModel.cs (hunk 1 above)
Comment (danroth27) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Models/ManageViewModels/IndexViewModel.cs]: We chatted in person and agreed to keep this property on the view model

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Startup.cs (hunk 2 above)
Comment (danroth27) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Startup.cs]: Removed

**Comment by danroth27** [comment]
@phenning let's get this merged

**Pull request merged**